### PR TITLE
Add Courts in Malaysia

### PR DESCRIPTION
**Pull request by MASangys**
https://en.wikipedia.org/wiki/Courts_Malaysia
https://www.courts.com.my/

### CODE REVIEW

File: brands/shop/furniture.json [hunk 1]
```diff
@@ -170,6 +170,18 @@
       "shop": "furniture"
     }
   },
+  "shop/furniture|  "Courts Malaysia": {
```
Comment (bhousel) [inline brands/shop/furniture.json:173]: Whoa looks like an extra quote here is messing things up.

```suggestion
  "shop/furniture|Courts Malaysia": {
```

**Comment by bhousel** [comment]
Looks good now

**Comment by MASangys** [comment]
However, there are another Courts in Singapore woth different Wikidata, so I added this as Courts Malaysia to differenciate between them

**Comment by bhousel** [comment]
> However, there are another Courts in Singapore woth different Wikidata, so I added this as Courts Malaysia to differenciate between them

oh weird - it probably should just be named "Courts"

**Comment by MASangys** [comment]
Then we should ask the editor at Wikipedia. because they do it,  I just follow their rules,  but commonly this tag for Courts should be apply for multi countries also.

**Comment by bhousel** [comment]
For now I just added 2 entries, one for Singapore and one for Malaysia - c98fc281

**Pull request merged**